data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/stream/ShardConsumer.java

```diff
@@ -289,7 +303,6 @@ public void run() {
                         continue;
                     }
                     if (waitForExport) {
-                        checkpointer.checkpoint(sequenceNumber);
                         waitForExport();
                         waitForExport = false;
                     }
```

Comment:
We'll want acknowledgment sets to be created here in batches from GetRecords calls within the shards.